include/OpenColorIO/OpenColorTypes.h

```diff
@@ -449,11 +449,14 @@ enum GpuLanguage
     GPU_LANGUAGE_GLSL_1_2,          ///< OpenGL Shading Language
     GPU_LANGUAGE_GLSL_1_3,          ///< OpenGL Shading Language
     GPU_LANGUAGE_GLSL_4_0,          ///< OpenGL Shading Language
-    GPU_LANGUAGE_HLSL_DX11,         ///< DirectX Shading Language
+    GPU_LANGUAGE_HLSL_SM_5_0,       ///< DirectX High Level Shading Language
     LANGUAGE_OSL_1,                 ///< Open Shading Language
     GPU_LANGUAGE_GLSL_ES_1_0,       ///< OpenGL ES Shading Language
     GPU_LANGUAGE_GLSL_ES_3_0,       ///< OpenGL ES Shading Language
-    GPU_LANGUAGE_MSL_2_0            ///< Metal Shading Language
+    GPU_LANGUAGE_MSL_2_0,           ///< Metal Shading Language
+
+    // Deprecated enum(s)
+    GPU_LANGUAGE_HLSL_DX11 = GPU_LANGUAGE_HLSL_SM_5_0
```

Comment:
Don't remember exactly but didn't you mention the use of OCIO_DEPRECATED macro? Would it work here? Looking at the attribute [[depracated]] I think it should but not sure about the other fallback methods.

Reply:
Removed my previous comment about this because I thought it might be needlessly disruptive.

If we did want it however, it would require modifications to the macro for MSVC: the fix for `__cplusplus` in Aras P's other PR, and the deletion of `__declspec(deprecated(msg))` which doesn't work with enums.